### DISCUSSION TITLE
Can you make changes to inbetweens within a keyframe

### DISCUSSION THREAD

**Discussion by MustardMonday** [keiwando/toonsquid#387] [Q&A]
It's hard to describe exactly what I mean, but I'll try.
When you make a keyframe it automates the inbetweens, and I'm wondering if you can edit those automated inbetweens like you could any other frame without effecting any key frames or other tweens. 

For example if I wanted to do a shot that zooms in on someone waving I'd want to keyframe it so it's smoother, would I be able to edit the individual tweens to make the persons arm wave? Because that would make it much easier to tell with perspective and things rather than simply making a whole new layer and having to adjust the size each time because of the zoom. Is that possible??

Hard to explain, hope this makes sense?

**Comment by keiwando** [comment]
You can't manually edit the keyframe values on frames between two keyframes, but if you want to create more complicated movements, you can chain multiple transforms together in a [hierarchy](https://toonsquid.com/handbook/transform/hierarchy/) to create more complex transform animations.

In your arm waving example it actually sounds more like you'd want to use a [camera layer](https://toonsquid.com/handbook/layers/camera/). You would animate the zoom on the camera layer and the arm waving on your character layer separately.
